README.md

```diff
@@ -565,3 +565,73 @@ const results = await testSheet.testRecords(inputRows)
 expect(results).toMatchObject(expectedOutputRows)
 ```
 
+## Using git and github to collaborate with Flatfile
+
+The platform-sdk allows you to use git to manage your Flatfile config.  You can also use git to get feedback on your implementation from Flatfile engineers.  Using git can be tricky, we recommend you follow the prescribed pattern to securely share your code with Flatfile.  When you share code in this way Flatfile can much more quickly diagnose problems you are having and provide guidance.  Flatfile greatly values interaction like this because it helps us shape the future of the platform-sdk and other features.  We see your code, admire your ingenuity, and see where you're struggling then we shape the platform-sdk to ease those rough spots
+
+Steps
+- Clone the platform-sdk-starter
+` $ git clone git@github.com:FlatFilers/platform-sdk-starter.git`
```

Comment:
🔧 Commands with extraneous info like `$/#` I've found always end up tripping up beginners b/c they will copy-and-paste and the ask "why is this not working?"

Reply:
That is annoying for copy/paste.  
Between `shell` `bash` and `console` do you have a reccomendation?

https://stackoverflow.com/questions/20303826/how-to-highlight-bash-shell-commands-in-markdown

maybe only show the prompt in screenshots... Show the whole flow in one screenshot?

Reply:
My recommendation would be to use `bash` and just leave off any console related things -- commands only. Screenshots are obviously hard to "copy and paste" into my terminal (I'm lazy sometimes lol).